tox.ini

```diff
@@ -126,11 +128,11 @@ environment =
     ALLOW_PLAINTEXT_LISTENER=yes
 
 [testenv:apicheck]
-commands = pip install -U -r{toxinidir}/requirements/dev.txt
+commands = pip install --upgrade -r{toxinidir}/requirements/dev.txt
```

Comment:
`pip install -U` can be quite opaque to readers because pip install currently has six `--u*` options, including `--user`.

`pip install --upgrade` makes the intention clearer in code that others will read.

% `python3 -m pip install --help | grep "\-\-u"`
```
  --user                      Install to the Python user install directory for
  -U, --upgrade               Upgrade all specified packages to the newest
  --upgrade-strategy <upgrade_strategy>
  --use-pep517                Use PEP 517 for building source distributions
  --use-feature <feature>     Enable new functionality, that may be backward
  --use-deprecated <feature>  Enable deprecated functionality, that will be
```